persistence/validator.go

```diff
@@ -46,76 +47,53 @@ func (p PostgresContext) UpdateValidator(address []byte, serviceURL string, stak
 	})
 }
 
-func (p PostgresContext) GetValidatorStakeAmount(height int64, address []byte) (string, error) {
+func (p *PostgresContext) GetValidatorStakeAmount(height int64, address []byte) (string, error) {
 	return p.getActorStakeAmount(types.ValidatorActor, address, height)
 }
 
-func (p PostgresContext) SetValidatorStakeAmount(address []byte, stakeAmount string) error {
+func (p *PostgresContext) SetValidatorStakeAmount(address []byte, stakeAmount string) error {
 	return p.setActorStakeAmount(types.ValidatorActor, address, stakeAmount)
 }
 
-func (p PostgresContext) GetValidatorsReadyToUnstake(height int64, status int32) ([]modules.IUnstakingActor, error) {
+func (p *PostgresContext) GetValidatorsReadyToUnstake(height int64, status int32) ([]modules.IUnstakingActor, error) {
 	return p.GetActorsReadyToUnstake(types.ValidatorActor, height)
 }
 
-func (p PostgresContext) GetValidatorStatus(address []byte, height int64) (int32, error) {
+func (p *PostgresContext) GetValidatorStatus(address []byte, height int64) (int32, error) {
 	return p.GetActorStatus(types.ValidatorActor, address, height)
 }
 
-func (p PostgresContext) SetValidatorUnstakingHeightAndStatus(address []byte, unstakingHeight int64, status int32) error {
+func (p *PostgresContext) SetValidatorUnstakingHeightAndStatus(address []byte, unstakingHeight int64, status int32) error {
 	return p.SetActorUnstakingHeightAndStatus(types.ValidatorActor, address, unstakingHeight)
 }
 
-func (p PostgresContext) GetValidatorPauseHeightIfExists(address []byte, height int64) (int64, error) {
+func (p *PostgresContext) GetValidatorPauseHeightIfExists(address []byte, height int64) (int64, error) {
 	return p.GetActorPauseHeightIfExists(types.ValidatorActor, address, height)
 }
 
-func (p PostgresContext) SetValidatorsStatusAndUnstakingHeightIfPausedBefore(pausedBeforeHeight, unstakingHeight int64, status int32) error {
+func (p *PostgresContext) SetValidatorsStatusAndUnstakingHeightIfPausedBefore(pausedBeforeHeight, unstakingHeight int64, status int32) error {
 	return p.SetActorStatusAndUnstakingHeightIfPausedBefore(types.ValidatorActor, pausedBeforeHeight, unstakingHeight)
 }
 
-func (p PostgresContext) SetValidatorPauseHeight(address []byte, height int64) error {
+func (p *PostgresContext) SetValidatorPauseHeight(address []byte, height int64) error {
 	return p.SetActorPauseHeight(types.ValidatorActor, address, height)
 }
 
-// TODO(team): The Get & Update operations need to be made atomic
-// TODO(team): Deprecate this functiona altogether and use UpdateValidator where applicable
-func (p PostgresContext) setValidatorStakedTokens(address []byte, tokens string) error {
```

Comment:
Was this function unused?

Reply:
@Olshansk yeah this was unused. Should I add it back?

Reply:
Nah, if it's locally scoped, usused, and not needed for documentation purposes, it's increasing cognitive load.

We'll add it back if we ever need it.